### ISSUE TITLE
Could not configure release stage on android

### ISSUE THREAD

**Issue by sm-kuniyoshi** [bugsnag/bugsnag-unity#123]
### Description

Android build could not configure `release stage`.  It goes `production` even if set to `Staging`.

### Issue

Use configured release stage.

### Environment

- macOS Mojave 10.14.2
- Unity 2018.3.3f1
- BugsnagUnity 4.1.1
- Galaxy S8, Android 8.0.0

### Reproduce

1. Download unity package from `https://github.com/bugsnag/bugsnag-unity/releases/tag/v4.1.1`
2. Create a new project with Unity 2018.3.3f1
3. Import `Bugsnag.unitypackage`
4. Switch platform to Android
5. Add Button UI
6. Define BugsnagTest script
7. Add BugsnagTest to the button, and set up api key, release stage, and on click event
8. Play the game, and click the button
9. The bugsnag reports the exception at release stage `Staging` exactly
10. Build android (Add Open Scenes, Set package name to `ProductName.DefaultCompany`)
11. Install, start, and click the button
12. Then bugsnag reports the exception at release stage `production` incrrectly

### Example code snippet

```cs
using System;
using BugsnagUnity;
using UnityEngine;


namespace DefaultNamespace
{

    public class TestBugsnag : MonoBehaviour
    {

        public string ApiKey;

        public string ReleaseStage;

        private void Awake()
        {
            Bugsnag.Init(ApiKey);
            Bugsnag.Configuration.AutoNotify = true;
            Bugsnag.Configuration.AutoCaptureSessions = true;
            Bugsnag.Configuration.UniqueLogsTimePeriod = TimeSpan.FromSeconds(5);
            Bugsnag.Configuration.NotifyLevel = LogType.Exception;
            Bugsnag.Configuration.MaximumBreadcrumbs = 25;
            Bugsnag.Configuration.NotifyReleaseStages = new[]
            {
                "Development",
                "Staging",
                "Production",
            };

            Bugsnag.Configuration.ReleaseStage = ReleaseStage;
        }

        public void CreateException()
        {
            var a = 0;
            var b = 2;
            var c = b / a;
        }

    }

}
```

**Comment by Cawllec** [comment]
Hi @sm-kuniyoshi thanks for the report, we're aware of the issue and working on a fix.

**Comment by kattrali** [comment]
Fixed by #124
